apps/builder/app/shared/instance-utils.ts

```diff
@@ -533,6 +547,17 @@ const traverseStyleValue = (
   value satisfies never;
 };
 
+type InstancesSlice = {
```

Comment:
as an idea: InstancesFragment in case you like it more

Reply:
More like slice of cheese with holes, fragment we use for component name already